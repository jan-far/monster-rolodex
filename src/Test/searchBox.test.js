import React from "react";
import { shallow } from "enzyme";
import SearchBox from "../Components/SearchBox";

describe("SearchBox Tests", () => {
  const mockPlaceholder = "Test suite";
  const mockHandleChange = jest.fn();

  it("make a snapshot of SearchBox comonent", () => {
    expect(
      shallow(<SearchBox placeholder={mockPlaceholder} handleChange={mockHandleChange} />)
    ).toMatchSnapshot();
  });
});
