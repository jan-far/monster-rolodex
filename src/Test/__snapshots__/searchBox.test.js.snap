// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchBox Tests make a snapshot of SearchBox comonent 1`] = `ShallowWrapper {}`;
